app/src/main/java/com/jalasoft/routesapp/ui/adminPages/routeEditor/view/RouteEditorFragment.kt

```diff
@@ -123,25 +231,47 @@ class RouteEditorFragment : Fragment(), OnMapReadyCallback, GoogleMap.OnMarkerCl
         map.animateCamera(cu)
     }
 
-    private fun addMarker(point: LatLng, withBitmap: Bitmap): Marker? {
-        return map.addMarker(MarkerOptions().position(point).icon(BitmapDescriptorFactory.fromBitmap(withBitmap)))
+    private fun addMarker(point: LatLng, withBitmap: Bitmap, isMarkerStop: Boolean): Marker? {
+        val mark = map.addMarker(MarkerOptions().position(point).icon(BitmapDescriptorFactory.fromBitmap(withBitmap)))
+        mark?.tag = isMarkerStop
+        return mark
     }
 
     override fun onMarkerClick(p0: Marker): Boolean {
```

Comment:
maybe you can rename this variable to something more descriptive

Reply:
Renamed